_dockerize/.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
No new line at the end of file.
The following setting could be added to vscode settings to automatically add it: `"files.insertFinalNewline": true`, or go to `Settings` and search for `insert final`